### PR TITLE
Fixedpoints with documentation

### PR DESCRIPTION
**Pull request by LucDupAtGitHub**
the title says it all

the original fixedpoints related PR will be removed

### CODE REVIEW

File: docs/.settings/org.scala-ide.sdt.core.prefs [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#Generated by sbteclipse
+#Sat Feb 20 20:08:22 CET 2016
+scala.compiler.additionalParams=-encoding UTF-8 -feature -language\:existentials -language\:higherKinds -language\:implicitConversions -language\:experimental.macros -Xlint -Yinline-warnings -Yno-adapted-args -Ywarn-numeric-widen -Ywarn-value-discard -Ywarn-unused-import
+Xplugin=/home/lucd/.ivy2/cache/org.scalamacros/paradise_2.11.7/jars/paradise_2.11.7-2.1.0-M5.jar,/home/lucd/.ivy2/cache/org.spire-math/kind-projector_2.11/jars/kind-projector_2.11-0.6.3.jar
+deprecation=true
+Xfatal-warnings=true
+Xfuture=true
+unchecked=true
+scala.compiler.useProjectSettings=true
+Ywarn-dead-code=true
```
Comment (DavidGregory084) [inline docs/.settings/org.scala-ide.sdt.core.prefs]: Looks like we need to add some eclipse-related stuff to the project .gitignore
```
.settings/
.project
.classpath
.loadpath
``

File: docs/src/main/tut/fix.md [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+      _.fold[String](showListAlgebra)
+
+    implicit def implicitShowList[Z]: Show[List[Z]] = new Show[List[Z]] {
+      def show(list: List[Z]): String = showList(list)
+    }
+```  
+
+## List Fixed Point example: Traverse
+
+So how do we *tarverse* lists.
```
Comment (DavidGregory084) [inline docs/src/main/tut/fix.md]: typo

Comment (LucDupAtGitHub) [inline docs/src/main/tut/fix.md]: I will fix stuff this evening (I am at work now)

thanks so much

**Comment by ceedubs** [comment]
Hi @LucDupAtGitHub. I'm sorry for the long silence on this one. I really appreciate the detailed documentation you have included with this change.

Having said that, I think the Typelevel incubator project [matryoshka](https://github.com/slamdata/matryoshka) already contains this representation of fixed points along with a lot more. I'd rather see fixed point effort going into matryoshka than splitting it between it and Cats. I'd imagine @sellout would be more than happy to have some documentation submitted to matryoshka if you'd like to transform this PR into a contribution there.

I'm going to go ahead and close this out. Let me know if there's a strong reason to add this functionality to Cats as opposed to matryoshka.

Thanks again.

**Pull request closed without merging**